### ISSUE TITLE
Enable Hallow spread.

### ISSUE THREAD

**Issue by LeoTheWo** [JavidPack/DisableCorruptionSpread#4]
To be honest, i dont mind Hallow at all. I actually like it. It would be nice if you'd add a way to stop both crimson/corruption from spreading but leave Hallow be.

**Comment by Isvoind** [comment]
I agree, to some extent you need the hallow for progression especially in modded playthroughs and for regular things like fishing. It would be nice if the default option was that hallow spread is disabled but you could type in the chat /enable_hallow or something to enable it.
